src/Codec/Xlsx/Parser/Stream.hs

```diff
@@ -688,7 +688,7 @@ parseStyle list =
 parseType :: SheetValues -> Either TypeError ExcelValueType
 parseType list =
   case findName "t" list of
-    Nothing -> pure Untyped
+    Nothing -> Right TN -- according to spec by default if `t` is missing then default value is `n` - number
```

Comment:
Maybe you could find a specific place in the spec talking about this? If no I could take a look a bit later

Reply:
Did I get you correctly that you propose to throw a reference to the spec here?
If so, I can do that no problems. Should I highlight it as well in comment for non-streamed parser?

Reply:
Yeah, section of the spec (or maybe reference to the schema?) would be great to prove the logic here or actually both places (or you could reference one from another)

Reply:
I am sorry, I placed wrong schema reference in the PR description (was reading specs in the first time
and was a bit confused by naming conventions). Now its clear to me what exactly to read :)

But I threw a correct ECMA references with paragraph numbers and quotes from `xsd`, because
could not find online format of that. Hope its fine!